src/main/java/net/pms/encoders/FFMpegVideo.java

```diff
@@ -1007,12 +1011,12 @@ public synchronized ProcessWrapper launchTranscode(
 					channels = configuration.getAudioChannelCount();
 				}
 
-				if (!customFFmpegOptions.contains("-ac ") && channels > 0) {
+				if (!customFFmpegOptions.contains("-ac ") && channels > 0 && params.aid.getAudioProperties().getNumberOfChannels() > configuration.getAudioChannelCount()) {
```

Comment:
Would it be better to use the `channels` variable instead of `configuration.getAudioChannelCount()` on this line?